src/graphql/resolvers/queries/anonymizedSession.js

```diff
@@ -13,14 +13,11 @@ export const fieldResolvers = {
 
       return null;
     },
+    tags: parent => {
+      dlog('tags');
 
-    speakers: parent => {
```

Comment:
why did we delete this?

Reply:
Speakers are not on AnonymizedSessions so the function wasn't needed for the resolver, it would never be called. It's not needed. I had added it error when I added that function to the different session types resolvers.